nautobot_chatops_ipfabric/worker.py

```diff
@@ -11,49 +11,118 @@
 from nautobot_chatops.workers import subcommand_of, handle_subcommands
 from netutils.ip import is_ip
 from netutils.mac import is_valid_mac
-from ipfabric_diagrams import IPFDiagram, Unicast
-from ipfabric import IPFClient
+from ipfabric_diagrams import Unicast
 
-from .ipfabric import IpFabric, LAST
+from .ipfabric_wrapper import (
+    IpFabric,
+    LAST,
+    DEFAULT_PAGE_LIMIT,
+)
 from .context import get_context, set_context
+from .utils import parse_hosts
 
 BASE_CMD = "ipfabric"
 IPFABRIC_LOGO_PATH = "ipfabric/ipfabric_logo.png"
 IPFABRIC_LOGO_ALT = "IPFabric Logo"
-
+CHATOPS_IPFABRIC = "nautobot_chatops_ipfabric"
+EMPTY = "(empty)"
+
+# URLs
+INVENTORY_DEVICES_URL = "tables/inventory/devices"
```

Comment:
I would prefer having this in the wrapper class

Reply:
agreed